com.pixelomer.ShijimaQt.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: com.pixelomer.ShijimaQt
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: shijima-qt
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --share=ipc
+  - --env=SHIJIMA_FLATPAK=1
+  - --socket=pulseaudio
+  - --env=QT_QPA_PLATFORM=xcb
+  # To be able to listen to and communicate with port 32456 for HTTP API and CLI
+  - --share=network
+  # Required for activating/deactivating shell extensions
+  - --talk-name=org.gnome.Shell
+  - --talk-name=org.kde.KWin
+  # Required for running gnome-extensions install
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
this require an expection. but why?

Reply:
The app installs a shell extension for tracking the active window's dimensions. On GNOME this is done with the gnome-extensions command. https://github.com/pixelomer/Shijima-Qt/blob/b41333d5dacdcd431bc35a2b0a2c032fcde06bc7/Platform/Linux/GNOME.cc#L58

Reply:
Is that an optional feature? Adding a link to extensions.gnome.org sounds a lot better than asking for flatpak-spawn.

Reply:
The application contains a zip file which it installs into GNOME using the gnome-extensions command. This extension isn't available on the GNOME Shell Extensions store/repository since it doesn't make sense to have this extension while not using Shijima-Qt.

It is possible to disable window tracking entirely. When it is disabled, shimeji will no longer be able to climb onto the active window and these permissions won't be required.

Reply:
> This extension isn't available on the GNOME Shell Extensions store/repository since it doesn't make sense to have this extension while not using Shijima-Qt.

There are lots of extensions published there that depends on an application, example https://flathub.org/apps/it.mijorus.smile and https://extensions.gnome.org/extension/6096/smile-complementary-extension/

It's not a good idea to install unreviewed extensions.

Reply:
I've disabled window tracking in [fcb1cb6](https://github.com/flathub/flathub/pull/6291/commits/fcb1cb62e93bb0cbb2f99bc96db3e18cb4960d55) for now, I can add it back in as an optional feature once the relevant changes have been made for it.